src/odemis/util/transform.py

```diff
@@ -431,6 +431,30 @@ def __call__(self, x):
         return self.apply(x)
     __call__.__doc__ = apply.__doc__
 
+    def fre(self, x, y):
+        """
+        Returns the RMS value of the fiducial error registration (FRE).
+
+        Parameters
+        ----------
+        x : (n, 2) array
+            Coordinates in the source reference frame.
+        y : (n, 2) array
+            Coordinates in the destination reference frame. Must be of same
+            dimensions as `x`.
+
+        Returns
+        -------
+        fre : float
+            The root mean squared fiducial registration error.
```

Comment:
Following on Thera's comment, please indicate that  the float >= 0, and that "the smaller, the better the fitting".

Reply:
I updated the docstring.